commands/check_cloudformation.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+import os
+from pathlib import Path
+
+import click
+
+from commands.bootstrap_cli import make_config
+from commands.cloudformation_checks.CheckCloudformationFailure import CheckCloudformationFailure
+from commands.cloudformation_checks.lint import lint
+from commands.copilot_cli import make_storage
+
+BASE_DIR = Path(__file__).parent.parent
+
+
+@click.group()
+def copilot():
+    pass
+
+
+def valid_checks():
+    # If you add something here, you need to remember to update the valid check docstring in check_cloudformation()
+    return {
+        "lint": lint,
+    }
+
+
+@click.group(invoke_without_command=True)
+@click.argument("checks", nargs=-1)
+@click.pass_context
+def check_cloudformation(ctx, checks):
+    """Runs the specific CHECK.
+
+    Valid checks are: all and lint
+    """
+
+    if len(checks) == 0:
+        checks = valid_checks()
+        running_checks = "all"
+    else:
+        running_checks = ' & '.join(', '.join(checks).rsplit(', ', 1))
+
+    for check in checks:
+        if check not in valid_checks():
+            click.secho(f"""Invalid check requested "{check}" """, fg="red")
```

Comment:
```suggestion
            click.secho(f"Invalid check requested {check}", fg="red")
```

Reply:
I wanted to keep the check name in quotes so it's more obvious which bit of the text it is...

![image](https://github.com/uktrade/copilot-tools/assets/8738245/5b19cb48-36a6-489f-aeab-09b5f5b4003c)

Reply:
In that case... I reckon we have two options.

```suggestion
            click.secho(f'Invalid check requested "{check}"', fg="red")
```

```suggestion
            click.secho(f"Invalid check requested \"{check}\"", fg="red")
```

Reply:
What's wrong with using triple quotes so that we don't have to escape the quotes in the string?

Reply:
Not sure, but I don't think it's the Pythonic way since triple quotes are primarily used to indicate Docstrings. How about my first option? No escaping involved there.

Reply:
Being able to declare a string like that and not mess about escaping quotes or changing the quotes to get around needing to escape them is a something I like about Python

I agree that triple quotes are used for Docstrings.

I cannot find anything saying it is unPythonic to use them for other strings, quite the opposite in fact.

Reply:
PEP8 doesn't have much to say except avoid slashes and be consistent: https://peps.python.org/pep-0008/#string-quotes